### PR TITLE
taxonomy: add some structure to data quality

### PR DESCRIPTION
**Pull request by alexgarel**
Add some structure to data quality.

Linked to #7718 and to PR #8032

### CODE REVIEW

File: taxonomies/data_quality.txt [hunk 1]
```diff
@@ -3,806 +3,842 @@
 ## --------------- Start with data quality errors. See: https://world.openfoodfacts.org/data-quality-errors
 
 # Data quality errors should not contain false positive, as opposed to data quality warnings
+en:Data quality errors
+description:en: Error detected on data
 
+<en:Data quality errors
+en:Nutrition errors
+description:en:Error on nutrition data (nutriments)
```
Comment (CharlesNepote) [inline taxonomies/data_quality.txt:11]:
```suggestion
description:en:Error on nutrition data (nutrients)
```

**Review by CharlesNepote — CHANGES_REQUESTED** [review]
Great, thanks.

File: taxonomies/data_quality.txt [hunk 2]
```diff
@@ -811,1221 +847,1638 @@ en:Packaging data complete
...
 #description:en:
+fix_action:en: add_quantity
 
 
 
 ### Fibers
 
+<en:Data quality warnings
+en:Fiber warnings
+description:en: Their might be an issue with fiber nutriment
```
Comment (stephanegigandet) [inline taxonomies/data_quality.txt:893]:
```suggestion
description:en: There might be an issue with the fibers nutrient
```

File: taxonomies/data_quality.txt [hunk 3]
```diff
@@ -811,1221 +847,1638 @@ en:Packaging data complete
...
 en:Serving quantity defined but quantity undefined
 #description:en:
+fix_action:en: add_quantity
 
 
 
 ### Fibers
 
+<en:Data quality warnings
+en:Fiber warnings
```
Comment (stephanegigandet) [inline taxonomies/data_quality.txt:892]:
```suggestion
en:Fibers warnings
```

File: taxonomies/data_quality.txt [hunk 4]
```diff
@@ -811,1221 +847,1638 @@ en:Packaging data complete
 
 #### Food groups
 
+<Data quality warnings
+en:Food groups warnings
+description:en: Food groups can't be infered
```
Comment (diivi) [inline taxonomies/data_quality.txt:852]:
```suggestion
description:en: Food groups can't be inferred
```

File: taxonomies/data_quality.txt [hunk 5]
```diff
@@ -2263,11 +2802,17 @@ en:Serving size in mg
...
+
+<en:Carbon footbrint warnings
 en:Carbon footprint from known ingredients but not from meat or fish
 #description:en:
 
+<en:Carbon footbrint warnings
 en:Carbon footprint from known ingredients less than from meat or fish
 #description:en:
 
+<en:Carbon footbrint warnings
```
[the remark this answers is not in the document]
Comment (alexgarel) [reply taxonomies/data_quality.txt:2816]: :sweat_smile: thanks @diivi !

**Comment by alexgarel** [comment]
Thank you everybody for the attentive reviews ! @CharlesNepote can you approve ?

**Review by CharlesNepote — APPROVED** [review]

**Pull request merged**